src/components/input-field/input-field.scss

```diff
@@ -65,7 +65,14 @@
 }
 
 .lime-looks-like-input-value {
-    @include shared_input-select-picker.looks-like-input-label;
+    @include shared_input-select-picker.looks-like-input-value;
+
+    .mdc-text-field--disabled & {
+        @include shared_input-select-picker.looks-disabled;
+    }
+}
+
+.lime-formatted-input {
```

Comment:
The disabled-styling felt like it belonged in `.lime-looks-like-input-value`, but the rest of the stuff here, `display: none` and stuff, doesn't belong there. That's specific to the formatted value use case.

Reply:
completely right!